Sources/SwiftDocC/Semantics/MarkupReferenceResolver.swift

```diff
@@ -71,6 +71,8 @@ struct MarkupReferenceResolver: MarkupRewriter {
             }
             
             // FIXME: Provide near-miss suggestion here. The user is likely to make mistakes with capitalization because of character input (rdar://59660520).
```

Comment:
```suggestion
```

Reply:
Or have I misunderstood? Either way this needs clarification.

Reply:
I don't consider the FIXME's (and rdar://59660520) fully resolved until the near-miss suggestions are structured as as `Solution` elements on the `Diagnostic`. 

With these changes the near-miss suggestions are part of the error message which both means that they're not presented as fix-its and that they need to be a single line to appear in inline error messages.

Solving that _may_ require breaking changes—since `TopicReferenceResolutionResult` is a public type. I'm not sure what the best solution is here since the resolver doesn't know the `URL` or `SourceRange` which it needs to create `Solution` elements with a `Replacement`.

I can add more info to the existing `FIXME` comment to better describe what's remaining. I could also consider rdar://59660520 done because the information is there and open a new issue for the remaining work to structure the near-miss suggestions are `Solution` elements with a `Replacement`. Which do you prefer?

Reply:
I created a new issue here https://github.com/apple/swift-docc/issues/438

Reply:
Updated in https://github.com/apple/swift-docc/pull/420/commits/9a6fa1254595241a54deb8811d18a159bf86b86f